frontend/src/views/web/contact-us.html

```diff
@@ -0,0 +1,60 @@
+<div class="white-bg view-container">
+    <main-header></main-header>
+    
+    <div class="grad-container">
+        <div class="row">
+            <div class="col s12 m8 l8">
+                    <br><br>
+                    <h4 align="center" class="text-dark-black"><strong>Contact Us</strong></h4>
+                    <form name="contactusForm" ng-submit="" id="contact-us" novalidate>
+        			<!-- Name -->
```

Comment:
@Ram81 looks like the code between line 10-38 is not indented properly. Please use 4 space indentation.